docs/reference/pip_install.rst

```diff
@@ -89,6 +93,25 @@ Some Examples:
   Don't use single or double quotes in a ``requirements.txt`` file.
 
 
+.. _`Per-requirement Overrides`:
+
+Per-requirement Overrides
++++++++++++++++++++++++++
+
+It is possible to set ``--install-options`` and ``--global-options``
+for each requirement in the requirements file:
+
+ ::
+
+    FooProject >= 1.2 --install-options="--prefix='/usr/local'" \
```

Comment:
`--install-option`  (singluar, not plural)
`--global-option`  (singluar, not plural)